site/docs/api/Editor.md

```diff
@@ -100,4 +101,64 @@ const App = () => {
     </Editor>
   )
 }
+```
+
+
+### Override default event handlers
+Customize how the default event handlers are handled
+
+```tsx {9-40,46-48}
+import {
+  DefaultEventHandlers,
+  DefaultEventHandlersOptions,
+  Editor,
+  EditorStore,
+  NodeId
+} from '@craftjs/core'
+
+class CustomEventHandlers extends DefaultEventHandlers {
+  isMultiSelectEnabled: (e: MouseEvent) => boolean
+
+  constructor(options: DefaultEventHandlersOptions & { store: EditorStore }) {
+    super(options)
+    this.isMultiSelectEnabled = options.isMultiSelectEnabled
+  }
```

Comment:
You can remove these lines. The `DefaultEventHandlers` constructor accepts an arbitrary `options` parameter in the constructor.

Although we should make an improvement to the `DefaultEventHandlers` class to accept a generic for options:
```tsx
export class DefaultEventHandlers<O = {}> extends CoreEventHandlers<
  DefaultEventHandlersOptions & O
>
```

Then in the custom event handler, we could do this:
```tsx
type CustomEventHandlersOptions = {}; 

class CustomEventHandlers extends DefaultEventHandlers<CustomEventHandlersOptions> {}
```